images/node/Dockerfile.centos7

```diff
@@ -0,0 +1,29 @@
+#
+# This is an OpenShift Origin node image with integrated OpenvSwitch SDN
+# If you do not require OVS SDN use the openshift/origin image instead.
+#
+# This image expects to have a volume mounted at /etc/origin/node that contains
+# a KUBECONFIG file giving the node permission to talk to the master and a
+# node configuration file.
+#
+# The standard name for this image is openshift/node
+#
+FROM openshift/origin
+
+COPY scripts/* /usr/local/bin/
+
+RUN INSTALL_PKGS="origin-sdn-ovs libmnl libnetfilter_conntrack openvswitch \
```

Comment:
origin-sdn-ovs itself is a few KiB, but it pulls in openvswitch and all of its dependencies. openshift3/node is 689.1MB, openshift/ose is 483.1MB.